DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Centre/CentreRankingViewModelTests.cs

```diff
@@ -28,7 +28,7 @@ public class CentreRankingViewModelTests
         public void CentreRankingViewModel_populates_expected_values_from_centre_ranks_with_centre_in_top_ten()
         {
             // When
-            var result = new CentreRankingViewModel(centreRankings, 3);
+            var result = new CentreRankingViewModel(centreRankings, 3, null, null);

```

Comment:
We can set the default values to " = null" in the view model so we don't need to specify nulls when creating this class